packages/commonwealth/client/scripts/views/components/component_kit/cw_text_input.tsx

```diff
@@ -137,6 +137,7 @@ export const CWTextInput = (props: TextInputProps) => {
     displayOnly,
     manualStatusMessage = '',
     manualValidationStatus = '',
+    validationStatus,
   } = props;
```

Comment:
I know you didn't write this file but cant we destructure the props in the params rather than on their own?